ui/src/views/Monitor/index.vue

```diff
@@ -4,9 +4,14 @@
 
 <script setup lang="ts">
 import { ref } from 'vue';
+import { useTerminalStore } from '@/store/modules/terminal.ts';
 import CustomTerminal from '@/components/CustomTerminal/index.vue';
 
+const terminalStore = useTerminalStore();
+
 const terminalType = ref<string>('common');
+
+terminalStore.setTerminalConfig('enableZmodem', false);
 </script>
 
 <style lang="scss" scoped>
```

Comment:
The above code seems to be an example of TypeScript setup using Vue with some custom components like `CustomTerminal`. It uses reactive property initialization for state management.

Suggestions:

  * The use of `$` symbol is deprecated since ES6, consider replacing it with `\`
```
$terminalType
```
  * If you plan on changing the content inside the `<script>` tag later,
you should ensure that all references (not only properties but anything else) match this new content.
```less
<script setup lang="ts">
// ...
</script>

Here's an updated version:
@@ -58,17 +63,5 @@

  * You might want to wrap your variables in single quotes if they're not meant to hold strings as part of a literal value
@lang scss
<style lang='scss' scoped>
@endlang.scss

This would look much cleaner and clearer to read.

Also,

If you have multiple instances that need different terminal type setting, you can keep them in separate files/components so each instance has its own configuration options
```typescript
<template>
   <div ref={this.$refs.container}>
      <!-- other stuff -->
   </div>
</template>

<script setup>
import TerminalComponent from './components/Terminal.vue'; // Use relative path

const containerRef = useRef();
</script>
```